### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by jvinis** [Ooples-Finance-LLC/OoplesFinance.StockIndicators#79]
Oscillator.cs
1) CalculateHighLowIndex
the hliList is empty and throws exception.

2) CalculateGatorOscillator
the alligatorList["Jaw"] should be alligatorList["Jaws"];

3) CalculateTechnicalRatings call with default parameters
when the CalculateAverageDirectionalIndex is called and GetInputValuesList is called raise an exception 
Calculations based off of IchimokuCloud can't be completed because this indicator doesn't have a single output.

**Comment by ooples** [comment]
@jvinis Can you please provide me with the information you used such as symbol, ohlc bars, etc
